spring-boot-admin-server-ui-javamelody/src/main/java/net/bull/javamelody/JavaMelodyService.java

```diff
@@ -0,0 +1,48 @@
+package net.bull.javamelody;
+
+import de.codecentric.boot.admin.event.ClientApplicationDeregisteredEvent;
+import de.codecentric.boot.admin.event.ClientApplicationRegisteredEvent;
+import de.codecentric.boot.admin.model.Application;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+/**
+ * Service class in the javamelody package, so we can access the javamelody Parameters class in which we can
+ * register and unregister applications. Using the {@link EventListener annotations} we receive notifications from service discovery.
+ */
+@Service
```

Comment:
Please don't rely on component scan - it might be disabled in the users' projects...